proto/osmosis/incentives/group.proto

```diff
@@ -14,7 +14,6 @@ enum SplittingPolicy {
   option (gogoproto.goproto_enum_prefix) = false;
 
   Volume = 0;
```

Comment:
Non-blocking question: How would you feel if we renamed this to something more specific e.g. `ByVolume`? I've noticed that the usage of this enum, `types.Volume`, feels somewhat underspecified as it could be referring to many things. I don't feel very strongly about this but thought I'd bring it up

Reply:
Agreed and changed